src/view/conditions/dateRange.jsx

```diff
@@ -97,6 +97,7 @@ export const formConfig = {
     // upon re-opening of the view, the UTC time can correctly be converted back to the selected
     // time zone and the selected time zone can be shown in the UI.
     const timezone = values.timezone || DEFAULT_TIMEZONE;
+    settings.timezone = timezone;
```

Comment:
Can you give your reasoning for this change please? It looks like we'll be emitting GMT to the Launch library unnecessarily.

After this was built, we added support for a remove transform (https://experienceleague.adobe.com/docs/launch/using/extension-dev/manifest.html#transforms), which would be the most appropriate way to keep `settings.timezone` out of the library at this point.

Reply:
Yes. We were emitting the `timezone` any time it wasn't the default value, and that seemed like a weird inconsistency when I was writing tests. I've added a commit to this PR to use a removeTransform in `extension.json` after speaking with Aaron about why I made the change.